.github/workflows/rumble-cve-data.yaml

```diff
@@ -0,0 +1,89 @@
+name: Generate Rumble CSV Data
+on:
+  workflow_dispatch:
+  push:
+    branches: [rumble-cve-workflow]
+
+env:
+  PROJECT_ID: "${{ secrets.PROJECT_ID }}"
+  STORAGE_BUCKET: "${{ secrets.STORAGE_BUCKET }}"
+  WORKLOAD_IDENTITY_PROVIDER: "${{ secrets.GCP_WORKLOAD_IDENTITY_PROVIDER }}"
+  SERVICE_ACCOUNT: "${{ secrets.GH_ACTION_SERVICE_ACCOUNT }}"
+  GH_TOKEN: ${{ github.token }}
+
+jobs:
+  generate-csvs:
+    runs-on: ubuntu-latest
+
+    strategy: 
+      matrix:
+        image: ["bash", "busybox", "deno", "git", "go", "kube-state-metrics", "mariadb", "maven", "minio", "minio-client", "nginx", "node", "php", "python", "rabbitmq", "ruby", "rust", "wait-for-it", "wolfi-base"]
```

Comment:
nit but what do you think about breaking this up on different lines? if we expand the number of image data we ingest it will get unwieldy

Reply:
Yeah it is something strange with GitHub actions - I tried breaking it up but it complains about an "unexpected end of the stream within a flow collection". I think we could move it into an environment variable though, and store the set of images as a variable in the repository settings here on GitHub?

Reply:
oh weird! because it's a data structure?

Reply:
Figured it out - apparently GitHub actions uses different kinds of YAML notation within a single file. The flow collection error message means that GitHub expects the matrix line to use that flow style. So then the question, what is flow style? TIL, it is a compact YAML notation that puts everything on a single line: https://www.yaml.info/learn/flowstyle.html